docs/audio.rst

```diff
@@ -18,17 +18,22 @@ Functions
 
     Play the source to completion.
 
-    ``source`` can be a built-in ``Sound`` from the ``microbit`` module or
-    an interable, each element of which must be an ``AudioFrame`` as described
+    :Param source: ``Sound`` or iterable of ``AudioFrame``
```

Comment:
This renders a bit weird:
![image](https://user-images.githubusercontent.com/29712657/100438641-5db26000-309a-11eb-8fd4-326432189c2d.png)

I'd recommend installing Sphinx locally (ideally with a VSCode plugging to see it side to side while editing) while trying all these things, otherwise the feedback loop by checking this in the CI is too long.

Reply:
Testing locally now. This looks better ![image](https://user-images.githubusercontent.com/31242877/100587459-7fa12200-32e8-11eb-8053-198c3aaf6949.png)

I think it's good to use the Sphinx syntax for parameters as we can then easily reference them elsewhere later should we want. It does use a lot of space on screen though, so maybe an ordered list is more appropriate.

Reply:
Nice! That does look much better.

> It does use a lot of space on screen though, so maybe an ordered list is more appropriate.

What would that look like?

Reply:
Less  space used, so I think this looks better and we aren't using reference links to parameters in the docs at the moment, so I think it's okay to do it like this
![image](https://user-images.githubusercontent.com/31242877/100846930-04ba4180-3477-11eb-8366-2fd14ce3a0ce.png)

Reply:
I have updated this in https://github.com/bbcmicrobit/micropython/pull/692/commits/b3c99574fadb428d8de669d2aa09d83074341c81